scripts/core.js

```diff
@@ -64,11 +66,13 @@ Hooks.on('init', () => {
   /*
   Initialize all Sub Modules
   */
-  Object.values(SUB_MODULES).forEach( (cl) => {
-    logger.info(COMMON.localize('SYB5E.Init.SubModule', {name: cl.NAME}));
+  Object.values(SUB_MODULES).forEach((cl) => {
+    logger.info(COMMON.localize('SYB5E.Init.SubModule', { name: cl.NAME }));
     cl.register();
   });
 
+  Journal.registerSheet('SYB5E', SymbaroumWide, { label: game.i18n.localize('SYB5E.journal.widejournal.name'), makeDefault: false });
```

Comment:
Move this registration to a 'register' function within 'SymbaroumWide', similar to the other registration functions.

Reply:
Resolved in update.